File: Coding/Python/on_all.py
def on_all ( eles ):
    new_eles = []
    for i in eles:
        new_eles.append(squares(i))
    return new_eles

def squares(num):
    square = num*num
    return square
        
elements = [1, 2, 3, 4, 5]
print elements
new_eles = on_all ( elements )
print new_eles

File: Coding/Ruby/on_all.rb
# Checks to see if a value is in the array
def on_all(eles)
    squares = []
    for i in eles
        squares.push(square(i))
    end
    return squares
end

def square(i)
    square = i*i
    return square
end

elements = [1,2,3,4,5]
puts("The original elements are #{elements}")
new_eles = on_all(elements)
puts("The new array is #{new_eles}")